java/gandiva/pom.xml

```diff
@@ -135,7 +135,7 @@
             <resource>
                 <directory>${gandiva.cpp.build.dir}</directory>
                 <includes>
-                    <include>irhelpers.bc</include>
+                    <include>src/gandiva/irhelpers.bc</include>
```

Comment:
currently we pass (build.dir)/(debug or release) as input to the gandiva.cpp.build.dir in travis

so this would have to be ../src/gandiva/irhelpers.bc

Reply:
Thanks. I've fixed.